price_service/server-rust/src/config.rs

```diff
@@ -0,0 +1,55 @@
+use {
+    libp2p::Multiaddr,
+    std::{
+        net::SocketAddr,
+        path::PathBuf,
+    },
+    structopt::StructOpt,
+};
+
+/// StructOpt definitions that provides the following arguments and commands:
+///
+/// Some of these arguments are not currently used, but are included for future use to guide the
+/// structure of the application.
+#[derive(StructOpt, Debug)]
+#[structopt(name = "pythnet", about = "PythNet")]
+pub enum Options {
+    /// Run the PythNet P2P service.
+    Run {
+        /// A Path to a protobuf encoded ed25519 private key.
+        #[structopt(short, long)]
+        id: Option<PathBuf>,
+
+        /// A Path to a protobuf encoded secp256k1 private key.
+        #[structopt(long)]
+        id_secp256k1: Option<PathBuf>,
```

Comment:
what are these keys for?

Reply:
possible leftover from when this started an actual validator?

Reply:
These were used when p2p was implemented in Rust, but passing them to the Go layer requires some additional Go wrappers that I haven't done yet. This PR is meant to be an initial structure and I think It's good to keep these to show how the software should be run even though they aren't forwarded through to Go yet. Instead Go is generating random keys/using hardcoded constants temporarily.